collectors/logsmanagement.plugin/plugin_logsmanagement.c

```diff
@@ -0,0 +1,411 @@
+        rrddim_set_by_pointer(  stats_chart_data->st_circ_buff_mem_uncompressed, 
+                                stats_chart_data->dim_circ_buff_mem_uncompressed_arr[i], 
+                                stats_chart_data->num_circ_buff_mem_uncompressed_arr[i]);
+        rrddim_set_by_pointer(  stats_chart_data->st_circ_buff_mem_compressed, 
+                                stats_chart_data->dim_circ_buff_mem_compressed_arr[i], 
+                                stats_chart_data->num_circ_buff_mem_compressed_arr[i]);
+        
+        /* Compression stats - update chart first time */
+        rrddim_set_by_pointer(  stats_chart_data->st_compression_ratio, 
+                                stats_chart_data->dim_compression_ratio[i], 
+                                stats_chart_data->num_compression_ratio_arr[i]);
+
+        /* DB disk usage stats - update chart first time */
+        rrddim_set_by_pointer(  stats_chart_data->st_disk_usage, 
+                                stats_chart_data->dim_disk_usage[i], 
+                                stats_chart_data->num_disk_usage_arr[i]);
+
+        chart_data_arr[i]->update(p_file_info, chart_data_arr[i], 1);
+
+        /* Custom charts - update chart first time */
+        for(int cus_off = 0; p_file_info->parser_cus_config[cus_off]; cus_off++){
+            rrddim_set_by_pointer(  chart_data_arr[i]->chart_data_cus_arr[cus_off]->st_cus,
+                                    chart_data_arr[i]->chart_data_cus_arr[cus_off]->dim_cus_count,
+                                    chart_data_arr[i]->chart_data_cus_arr[cus_off]->num_cus_count);
+        }
+        for(int cus_off = 0; p_file_info->parser_cus_config[cus_off]; cus_off++){
+            if(chart_data_arr[i]->chart_data_cus_arr[cus_off]->need_rrdset_done){
+                rrdset_done(chart_data_arr[i]->chart_data_cus_arr[cus_off]->st_cus);
+            }
+        }
+    }
+
+    // outside for loop as dimensions updated across different loop iterations, unlike chart_data_arr metrics.
+    rrdset_done(stats_chart_data->st_circ_buff_mem_total);
+    rrdset_done(stats_chart_data->st_circ_buff_mem_uncompressed);
+    rrdset_done(stats_chart_data->st_circ_buff_mem_compressed);
+    rrdset_done(stats_chart_data->st_compression_ratio);
+    rrdset_done(stats_chart_data->st_disk_usage);
+
+    usec_t step = g_logs_manag_update_every * USEC_PER_SEC;
+    heartbeat_t hb;
+    heartbeat_init(&hb);
+
+	while(!netdata_exit){
+
+        worker_is_idle();
+        (void) heartbeat_next(&hb, step);
+
+        if(unlikely(netdata_exit)) break;
+
+        rrdset_next(stats_chart_data->st_circ_buff_mem_total);
+        rrdset_next(stats_chart_data->st_circ_buff_mem_uncompressed);
+        rrdset_next(stats_chart_data->st_circ_buff_mem_compressed);
+        rrdset_next(stats_chart_data->st_compression_ratio);
+        rrdset_next(stats_chart_data->st_disk_usage);
```

Comment:
The comment above considered that chart creation would be here instead there. probably it would be the best to move for here the chart creation like we have in `cgroups`.